requirements/base.txt

```diff
@@ -3,19 +3,15 @@
 # needle (selenium is also used directly)
 nose==1.3.7
 Pillow==3.3.0
-
-# Note that to get setup.py to work correctly with needle we have
-# version constraints specified in setup.py as well. We will need
-# to change both to upgrade to selenium 3 or above.
-selenium==2.53.6
+selenium==3.3.1
 
 # Then the direct dependencies
 
 # For lazy evaluation of the page object's a11y_audit attribute
 lazy==1.2
 
 # For test assertions involving screenshots and visual diffs
-needle==0.4.1
+git+https://github.com/bfirsh/needle.git@2bc898654c5dc887d47a3ba26be92a6d42113440#egg=needle
```

Comment:
I hesitate to do this as we are not under control of the repo, and this commit could disappear. Acknowledging that the risk is pretty low, still I recommend that we either wait or fork the repo (either under your account if it's easier since it's temporary or the edx org).